### PR TITLE
 Removes the bugged wanted poster from metasec. 

### LINKED ISSUE #56915: Wanted Posters That Cheat And Spawn Are Bad
## Reproduction:

Saw this on Meta station in the main sec office. Had one spawn, and show as missing poster () for a `/obj/item/poster/wanted/missing`.

Not sure if its a set spawn or a loot table spawn off the top of my head, but well yeah, probably shouldn't really be spawning to begin with.

bad poster

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request
Fixes #56915
Adds a worse case senario to prevent the same sort of bug happening in the future. Fixes poster harddels
DON'T HOLD REFS WHEN THE THING YOU'RE REFFING DIES!!!

### CODE REVIEW

**Comment by SuperNovaa41** [comment]
I believe this fixes issue 

https://github.com/tgstation/tgstation/issues/56915

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -3454,6 +3454,17 @@
...
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer4,
+/obj/machinery/camera{
+	c_tag = "Science Hallway - Central";
+	dir = 4
+	},
+/obj/machinery/atmospherics/pipe/manifold/scrubbers/hidden/layer2{
+	dir = 8
+	},
+/turf/open/floor/iron/white,
+/area/science/research)
```
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm:3467]: uhm, old branch?

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/contraband.dm [hunk 2]
```diff
@@ -21,16 +21,25 @@
...
+		if(QDELETED(poster_structure))
+			stack_trace("A poster was initialized with a qdeleted poster_structure, something's gone wrong")
+			return INITIALIZE_HINT_QDEL
 		name = poster_structure.poster_item_name
 		desc = poster_structure.poster_item_desc
 		icon_state = poster_structure.poster_item_icon_state
 
 		name = "[name] - [poster_structure.original_name]"
+		//If the poster structure is being deleted something has gone wrong, kill yourself off too
+		RegisterSignal(poster_structure, COMSIG_PARENT_QDELETING, .proc/react_to_deletion)
```
Comment (coiax) [inline code/game/objects/effects/contraband.dm:33]: Does the parent structure have a reference to the poster?

**Review by Cyberboss — APPROVED** [review]

File: code/game/objects/effects/contraband.dm (hunk 2 above)
Comment (Cyberboss) [reply code/game/objects/effects/contraband.dm:33]: Quick search says no

**Pull request merged**